tests/Main.hs

```diff
@@ -365,15 +365,22 @@ sinkEntrySpec =
 loadEntrySpec :: SpecWith (Path Abs File)
 loadEntrySpec =
   context "when an entry is loaded" $
-    it "is there" $ \path -> property $ \m b s -> do
+    it "is there" $ \path -> property $ \m b s r -> do
       let vpath = deriveVacant path
       B.writeFile (toFilePath vpath) b
+      addUTCTime (-60) <$> liftIO getCurrentTime >>= setModificationTime vpath
```

Comment:
Err, please no. If modification time preservation is not requested (`r` is `False`), just don't mess with it at all and don't check either.